lint/doc/Changelog.md

```diff
@@ -0,0 +1,9 @@
+# Witnesslint Changelog
```

Comment:
@PhilippWendler would it make sense to add a SPDX header to the Changelog.md?

Reply:
All files need an SPDX header. And there should exist a CI job that enforces this (likely after the move to GitLab).

Reply:
Added headers for `Changelog.md` and `Developing.md`.